Resources/Maps/northway.yml

```
@@ -20329,6 +20402,14 @@
       linearDamping: 0
       canCollide: False
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)